### PR TITLE
mock torch: english doc

### PR DESCRIPTION
**Pull request by farmerzhang1**

### CODE REVIEW

File: en/docs/cookies/oneflow_torch.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+With the alignment of OneFlow API and PyTorch, users can easily migrate the model from PyTorch to OneFlow. In this article, we will introduce three methods to port PyTorch code to OneFlow.
+
+## 1. import oneflow as torch
+
+Change the original code ` import torch ` to:
+
+```py
+import oneflow as torch
+```
```
Comment (farmerzhang1) [inline en/docs/cookies/oneflow_torch.md:11]: 这个链接是啥

File: en/docs/cookies/oneflow_torch.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+https://gist.github.com/Artemisyuu/3185c4af8489ea72b4a09e7ad70b9acf
+
+You can train the original model with OneFlow; However, you have to manually modify all files that contain `import torch` by this approach. In addition, it is necessary to modify the source code if a third-party library uses `torch`.
```
Comment (farmerzhang1) [inline en/docs/cookies/oneflow_torch.md:14]: all files that -> all the files that?

File: en/docs/cookies/oneflow_torch.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+## 2. Using the command-line tool
+
+Oneflow provides a command-line tool, which helps to simulate the environment of PyTorch within OneFlow’s Python Package and forward references of Pytorch to the real OneFlow module. The specific steps are as follows:
+
+
+Enabling the simulation of PyTorch
```
Comment (farmerzhang1) [inline en/docs/cookies/oneflow_torch.md:21]: simulation要不要都改成mocking?

File: en/docs/cookies/oneflow_torch.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+```shell
+eval $(oneflow-mock-torch)
+```
+https://gist.github.com/Artemisyuu/c6bfb433d6fd477a9b865c5cf76cd388
```
Comment (farmerzhang1) [inline en/docs/cookies/oneflow_torch.md:26]: 链接

File: en/docs/cookies/oneflow_torch.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+or
+
+```shell
+eval $(python3 -m oneflow.mock_torch disable)
+```
+https://gist.github.com/Artemisyuu/6d35e06ac5c9a0ce70ea2d969871ea1c
+
+## 3. Using built-in functions of OneFlow
+
+The mock function with fine granularity provided by our company allows users to control whether to enable this function for a piece of code.
```
Comment (farmerzhang1) [inline en/docs/cookies/oneflow_torch.md:62]: 不是很通顺

File: en/docs/cookies/oneflow_torch.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+```py
+mock.enable()
+...
+with mock.disable()
+    module = torch.load_module(...)
+# train the module with oneflow
+```
+https://gist.github.com/Artemisyuu/f7a8f946a84c42a3249d6e3d5555a639
+
+A dictionary with the value of module has been saved separately in enable and disable mode. When enable or disable is turned on and off, the dictionary will replace `sys.modules` and the global variables that the current module belongs to. Therefore, users are required to `import` the module they need in each mode, and the code below will raise an error `name 'torch' is not defined`  in `with` statment of disable.
```
Comment (farmerzhang1) [inline en/docs/cookies/oneflow_torch.md:97]: has been saved -> is saved?

Comment (farmerzhang1) [reply en/docs/cookies/oneflow_torch.md:97]: `When enable or disable is turned on and off` 感觉有点怪

File: en/docs/cookies/oneflow_torch.md (hunk 1 above)
Comment (Artemisyuu) [reply en/docs/cookies/oneflow_torch.md:11]: 链接已删除

File: en/docs/cookies/oneflow_torch.md (hunk 2 above)
Comment (Artemisyuu) [inline en/docs/cookies/oneflow_torch.md]:
```suggestion
You can train the original model with OneFlow; however, you have to manually modify all the files that contain `import torch` by this approach. In addition, it is necessary to modify the source code if a third-party library uses `torch`.
```

File: en/docs/cookies/oneflow_torch.md (hunk 3 above)
Comment (Artemisyuu) [reply en/docs/cookies/oneflow_torch.md]: 已改

File: en/docs/cookies/oneflow_torch.md (hunk 5 above)
Comment (Artemisyuu) [inline en/docs/cookies/oneflow_torch.md]:
```suggestion
We provide the mock function with fine granularity, and the users can determine whether to enable this function for a piece of code.
```

File: en/docs/cookies/oneflow_torch.md (hunk 6 above)
Comment (Artemisyuu) [inline en/docs/cookies/oneflow_torch.md]:
```suggestion
A dictionary with the value of module is saved separately in enable and disable mode. When you turn enable or disable on and off, the dictionary will replace `sys.modules` and the global variables that the current module belongs to. Therefore, users are required to `import` the module they need in each mode, and the code below will raise an error `name 'torch' is not defined`  in `with` statment of disable.
```

File: en/docs/cookies/oneflow_torch.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
+# Oneflow is compatible with PyTorch
```
[the remark this answers is not in the document]
Comment (farmerzhang1) [reply en/docs/cookies/oneflow_torch.md:1]: 我把中文标题改成oneflow模拟pytorch了

[the remark this answers is not in the document]
Comment (farmerzhang1) [reply en/docs/cookies/oneflow_torch.md:1]: 还有，OneFlow的F应该是大写的

**Review by doombeaker — APPROVED** [review]
替杨婷 approve

**Pull request merged**